internal/k8s/leader_election_callbacks.go

```diff
@@ -15,24 +15,38 @@
 package k8s
 
 import (
+	"sync"
+
 	v1 "k8s.io/api/core/v1"
 
+	avicache "github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/internal/cache"
 	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/internal/lib"
+	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/internal/rest"
 	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/utils"
 )
 
+var once sync.Once
+
 func (c *AviController) OnStartedLeading() {
 	lib.AKOControlConfig().SetIsLeaderFlag(true)
 	utils.AviLog.Debugf("AKO became a leader")
 	lib.AKOControlConfig().PodEventf(v1.EventTypeNormal, "LeaderElection", "AKO became a leader")
 	c.publishAllParentVSKeysToRestLayer()
```

Comment:
@swathinsankaran: If deleteConfig is true, we still populate all keys to Rest layer?

Reply:
`FullSyncK8s` returns early without populating any keys when the `deleteConfig` is set. Hence we will not publish any keys to rest layer.
[internal/k8s/ako_init.go#L743](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/1afd4b5720c3f69d095903d1cd513e163228915b/internal/k8s/ako_init.go#L743)